website/www/site/content/en/contribute/postcommits-policies-details.md

```diff
@@ -50,7 +50,7 @@ situation, fixing failing tests is our highest priority.
 
 ## A flaky test is a critical/P1 issue {#flake_is_failing}
 
-Flaky tests are considered failing tests, and fixing a flaky test is a
+Flaky tests are considered as failing tests, and fixing a flaky test is a
```

Comment:
Sorry for the slow review here. I don't think this is more correct though, in general 'considered' probably shouldn't be followed by 'as' in cases like this

- https://ell.stackexchange.com/questions/83401/considered-vs-considered-as
- https://bringmethenews.com/life/instagrammar-please-please-stop-saying-considered-as